addons/l10n_eg_edi_eta/models/account_edi_format.py

```diff
@@ -148,7 +153,7 @@ def _cancel_invoice_edi_eta(self, invoice):
         }
 
     @api.model
-    def _l10n_eg_get_einvoice_status(self, invoice):
+    def _l10n_eg_get_einvoice_document_summary(self, invoice):
```

Comment:
due to the refactor, the function now can return multiple data types 
line 158 returns a dict
line 167 returns a dict
line 170 returns a list

Reply:
I've updated the code to always return a dict.